### DISCUSSION TITLE
Nested mapping of list

### DISCUSSION THREAD

**Discussion by Mr-EmPee** [mapstruct/mapstruct#3789] [Q&A]
Hi, I would like to map a nested list to a list of another type but it gives the error:
"error: Can't generate mapping method from non-iterable type to iterable type from java stdlib."

This is my code:
```java
  @Mapping(source = "ids", target = ".")
  List<UUID> map(Foo foo);

  class Foo {
    List<String> ids = List.of(UUID.randomUUID().toString(), UUID.randomUUID().toString());
  }
```

**Comment by filiphr** [comment]
The best solution for something like this would be to provide custom method for the mapping. MapStruct has no automatic way to to this mapping.
